README.md

```diff
@@ -133,3 +133,5 @@ Some examples of what to say
 - when is sunset
 - what state is the sun
 - how long until dusk
+
+- whats the ip of the kitchen light
```

Comment:
what's? I don't prented to correct your english though

Reply:
agreed, However i usually try to it both ways if i remember because when using the TXT input widget the ' is not all ways typed and fails to recognise as a result